.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Purpose
+_Describe the problem or feature in addition to a link to the issues._
+
+## Approach
+_How does this change address the problem?_
+
+## Tests for New Behavior
+_What new tests were added to cover new features or behaviors?_
+
+## Checklist
+_Make sure your change addresses the following concerns, if applicable._
+- [ ] Did you add tests for any new features or behaviors?
+- [ ] Did you run ``tox -e flake8`` to make sure code style is consistent?
+- [ ] Did you build and review the docs?
```

Comment:
What you and I often do is throw up a link where the reviewer can see the built docs. Can we encourage people to do this (especially if doc changes are significant), or do you think that will be too cumbersome?